pages/docs/concepts/protocol.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Protocol
+weight: 25
+---
+
+
+## What is a protocol?
+A protocol is a mechanism that handles the exchange of messages. It provides a way of getting messages from one point in an event-driven architecture to another. A protocol could either be wired or API.
```

Comment:
- protocols are not only in context of the event driven architectures
- what do you mean protocol could be an API?
- I'm not sure `mechanism` is the correct word. A protocol is nothing else than a set of rules that are used to specify how information should be exchanged between applications and/or servers